README.md

```diff
@@ -1,71 +1,98 @@
-[![Publish Pipeline Badge](https://img.shields.io/endpoint?url=https://winget-pme.azurefd.net/api/GetServiceComponentStatusBadge?component=PublishPipeline "Publish Pipeline Badge")](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=338)
+[![Validation Pipeline Badge](https://img.shields.io/endpoint?url=https://winget-pme.azurefd.net/api/GetServiceComponentStatusBadge?component=ValidationPipeline 'Validation Pipeline Badge')](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=337)
+[![Publish Pipeline Badge](https://img.shields.io/endpoint?url=https://winget-pme.azurefd.net/api/GetServiceComponentStatusBadge?component=PublishPipeline 'Publish Pipeline Badge')](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=338)
 [![Github Status](https://img.shields.io/endpoint?url=https://api.bittu.eu.org/github-status-badge-endpoint)](https://www.githubstatus.com)
 
 This repository contains the manifest files for the **Windows Package Manager** default source. You are highly encouraged to submit manifests for your favorite application.
+
 > Note: At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.
 
 The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring third-party repositories offering private winget package hosting, see [third-party repositories](THIRD_PARTY.md).
 
 # Submitting a Package
+
 To submit a package to this repository, you should follow these steps:
-1) Follow the [Contributing](#Contributing) guidelines below.
-2) [Author](AUTHORING_MANIFESTS.md) a Manifest.
-3) [Test](#test-your-manifest) your manifest.
-4) [Submit](#submit-your-pr) your pull request (PR).
-5) Respond to any feedback in your PR.
+
+1. Follow the [Contributing](#Contributing) guidelines below.
+2. [Author](AUTHORING_MANIFESTS.md) a Manifest.
+3. [Test](#test-your-manifest) your manifest.
+4. [Submit](#submit-your-pr) your pull request (PR).
+5. Respond to any feedback in your PR.
 
 > Note: Please check the package's manifest you intend to submit does not already exist in the repository, and there are no open PRs for it in order to avoid duplicates.
 
 ## Authoring a Manifest
 
-You may either use the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create), the [YamlCreate](#using-the-yamlcreateps1) PowerShell script, or you can craft a manifest manually following our [authoring guidelines](AUTHORING_MANIFESTS.md).
+There are some tools available to help you author a manifest.
+
+- [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create)
+- [Komac - **Ko**tlin **Ma**nifest **C**reator for WinGet](https://github.com/russellbanks/Komac)
```

Comment:
I had a conversation with @denelon before opening this PR. My thoughts are we should list software in README with what extra functionality/advantages they have above already existing tools. 

We would certainly not add a project to the README that just creates manifests with only mandatory fields with some user input prompts here and there, without any additional metadata. I think we can add a note on it that the tool should stand apart from other tools and should not do the same thing that other tools already do.

Reply:
I agree with @vedantmgoyal2009 in this respect - I've put near enough 1000 hours into writing Komac as a hobby project; I seriously doubt more like it will just crop up by adding it to some documentation here.

Reply:
@Trenly I had this same concern when I was chatting with @vedantmgoyal2009. I don't want to create some kind of contest for generating additional tools to get the same job done. I'm more than happy to have some kind of mention or reference to other projects, and clearly the work @russellbanks has done is in another language (so it's not something that would be easy to add to wingetcreate or YAMLCreate) which other developers may prefer, but I don't want to end up with a list of competing projects.